client/buildpacks.go

```diff
@@ -47,6 +47,7 @@ type buildpacksFunctionServer struct {
 	logStderr          *os.File
 	stdoutFile         string
 	stderrFile         string
+	functionConcurrency	   uint
```

Comment:
I think we may want to hold off on merging this for a little while longer.

Orrrr, maybe we should add a generic new parameter called like `envs` that takes a map/list of additional env vars passed to your docker container? This might be a good time to do that because I suspect we'll have more env vars in the future and also not every language will need the FUNCTION_CONCURRENCY env var specified.

Wdyt?

Reply:
That sounds like a good idea. I will work on the implementation